src/features/game/expansion/Game.tsx

```diff
@@ -100,9 +100,15 @@ export const Game: React.FC = () => {
         </Panel>
       </Modal>
 
-      <Water level={gameState.context.state.expansions.length + 1} />
-      <Land />
-      <Hud />
+      <div className="absolute z-0 w-full h-full">
+        <Water level={gameState.context.state.expansions.length + 1} />
+      </div>
+      <div className="absolute z-10 w-full h-full">
+        <Land />
+      </div>
+      <div className="absolute z-20">
+        <Hud />
+      </div>
```

Comment:
This explicitly sets the z-index for the Water, Land and Hud. This prevents elements on the Land and Water from overriding the Hud and other such malarkey.

![malarkey](https://user-images.githubusercontent.com/41215134/184060900-cbbb2271-2b17-43fa-a226-d8ffcdca04cd.png)

Thanks @adamhannigan !